### PR TITLE
Getting started with Sanity CMS

### LINKED ISSUE #6661: Getting Started with Sanity CMS
### Topic Suggestion
Getting Started with Sanity CMS.

#### Pre-submission advice
Introduction
Building applications and having to worry about the content or database is not efficient.

Sanity is an Open Source Content Management System built with React. Sanity allows you to focus on developing your application without having to worry about the content file storage and databases, Sanity will cover all the hard work for you and allow you to build scalable and user-friendly applications extremely easily. it is also versatile you can create simple things incredibly easily but if you need to create more complex queries and schemas you can do that as well.
 Sanity is a platform for structured content, it helps you manage and deliver your content easily. Another cool stuff about sanity is the Sanity Studio it's a dashboard where you can edit, update, collaborate and manage all of your content.

#### Key takeaways 
At the end of this tutorial the reader would fully understand:

1. CMS & JAM Stack Applications
2. Sanity Installation & Setup
3. Create a project with Sanity
4. Sanity Cli
5. Explore Sanity Studio
6. Schema Types
7. Query Data 
8. Deploy Sanity Studio

#### Article quality
In this tutorial, we are going to be taking a deep dive looking into CMS, JAM Stack Applications Sanity CMS, Setup, data querying, and lot. It's completely beginner-friendly

#### References
https://www.sanity.io/

### PR DESCRIPTION
**Pull request by abimbolataofeek**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #6661

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by abimbolataofeek** [comment]
Hi good morning @ahmadmardeni1, please my article is yet to be assigned for a while now please can you help me look into it.

**Comment by abimbolataofeek** [comment]
Hi good morning @hectorkambow , please my article is yet to be assigned for a while now please can you help me look into it.

**Comment by ahmadmardeni1** [comment]
Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept.
Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.
Be sure to see our resources page to see more info on plagiarism and what is considered as such.

[Getting started with Sanity CMS #6825](https://github.com/section-engineering-education/engineering-education/files/8193350/New.Scan.7_26.PM.Copyleaks.Report.pdf)

**Comment by abimbolataofeek** [comment]
plagiarism checked

**Comment by abimbolataofeek** [comment]
Good morning @ahmadmardeni1  please can you look into my pr

**Comment by lalith1403** [comment]
PFA the plagiarism report. It is higher than what we accept. Please make the changes accordingly. Thank you.
[New Scan 4_11 PM Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/8225133/New.Scan.4_11.PM.Copyleaks.Report.pdf)

**Comment by abimbolataofeek** [comment]
Plagiarism issue fixed @lalith1403

**Comment by abimbolataofeek** [comment]
Good morning @lalith1403  please can you look into my PR.

File: content/articles/getting-started-with-sanity-cms/index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+#### Sanity Command Line Interface
+We have arrived at the Sanity command-line interface, and we will need to answer a few questions so that we can create the ideal project structure.
+##### steps:
+- Sign in with Google, GitHub, or an email address and password.
+- Fill in the name of your project.
+- Use the dataset's default settings.
+- Select a project route.
+- Choose a blank project with no predefined schema to start sanity
+#### Starting sanity content studio
+- Run `sanity start` in your terminal.
```
Comment (Jethro-magaji) [inline content/articles/getting-started-with-sanity-cms/index.md:28]: All commands should be in "Bash" backticks

File: content/articles/getting-started-with-sanity-cms/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Sanity is a content management and delivery platform that makes it simple to manage and provide structured information in our database. With the Sanity studio dashboard, we can edit, update, contribute, and manage all of our content.
+#### Prerequisites
+- Install Node.js 
+- Code Editor (Visual Studio Code is recommended)
+### Getting Started
+#### Setting up our Sanity application - Sanity CLI Installation
+To get started with Sanity, install Sanity CLI.
+
+Run the command below in your code editior.
+- `npm install -g @sanity/cli`
```
Comment (Jethro-magaji) [inline content/articles/getting-started-with-sanity-cms/index.md:17]: This should be in "Bash"

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]
Hi! @abimbolataofeek 
Well done for such an amazing article. However, there are some corrections you need to make.
Run your article on tools like Grammarly and ProWritingAid for better editing, this would save the time to review your work.
Here's a [link](https://github.com/section-engineering-education/engineering-education/pull/6210/commits/acc44893d4ec258f4e077b9ce8e3e2ca2b9d6a9f) to a PR with little or no revision to guide you on formatting. However, if you have any questions or need help, you can reach out to me on the EngEd Slack channel.
Thanks.

**Comment by abimbolataofeek** [comment]
@Jethro-magaji  all correction has been fixed

**Comment by hectorkambow** [comment]
Please be sure to proofread the entire article as Peer Reviewers do not provide revisions services (and may not point out all pending issues).
If any other edits are required @Jethro-magaji we can close the PR to allow the student the proper time to make the edits.

**Comment by hectorkambow** [comment]
@abimbolataofeek

**Comment by hectorkambow** [comment]
any update on this PR are any other edits being made? @abimbolataofeek 

cc @Jethro-magaji

**Comment by abimbolataofeek** [comment]
@hectorkambow all corrections has been made, waiting for review

**Review by Jethro-magaji — APPROVED** [review]
Hi! @abimbolataofeek 
Well done for coming up with such an amazing article. Please take note of these corrections to minimize revision next time.
Thanks.

**Comment by abimbolataofeek** [comment]
@Jethro-magaji, thank you for reviewing my PR; I've learned from the mistakes you've pointed out.

**Comment by abimbolataofeek** [comment]
Hello, nothing is happening with my PR.

**Comment by hectorkambow** [comment]
Our team is going through the queue in the order of oldest to latest 🚀  

We will be sure to review asap

**Pull request merged**